workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowInstancePage.tsx

```diff
@@ -222,6 +236,43 @@ export const WorkflowInstancePage = ({
     navigate(urlToNavigate);
   }, [value, navigate, executeWorkflowLink]);
 
+  const handleRetrigger = async () => {
+    if (value) {
+      setIsRetrigger(true);
+      try {
+        await orchestratorApi.retriggerInstance(
+          value.instance.processId,
+          value.instance.id,
+        );
+        restart();
+      } catch (retriggerInstanceError) {
+        if (retriggerInstanceError.toString().includes('Failed Node Id')) {
+          setRetriggerError(`Run failed again`);
```

Comment:
Nit:
```suggestion
          setRetriggerError('Run failed again');
```

(the template strings are not needed here)